src/lib.rs

```diff
@@ -1653,6 +1653,96 @@ impl Bindings {
             ))
         }
     }
+    
+    /// Write these bindings as source text to stdout.
+    pub fn write_to_stdout(&self) -> io::Result<()> {
+        let bindings = self.to_string();
+        
+        match self.rustfmt_generated_string(bindings) {
+            Ok(Some((rustfmt_bindings, rustfmt_stderr))) => { 
+                println!("{}", rustfmt_bindings);
+                eprintln!("{}", rustfmt_stderr);
+            },
+            Ok(None) => (),
+            Err(err) => println!("{:?}", err),
+        }
+        
+        Ok(())
+    }
+    
+    /// Checks if rustfmt_bindings is set and runs rustfmt on the string
+    fn rustfmt_generated_string(&self, source: String) -> io::Result<Option<(String, String)>> {
```

Comment:
I think we can just let the child's `stderr` inherit our `stderr`, and that will let us simplify this function and its signature a bit.

We should end up with:

```rust
fn rustfmt_generated_string(&self, source: String) -> io::Result<String> { ... }
```

Reply:
Hey @fitzgen, if we inherit stderr, how can we match on the status code of rustfmt to perform the respective actions? Just let the stderr display rustfmt's errors?

Reply:
We inherit `stderr` by simply not calling `.stderr(..)` on the command that is being built. It doesn't affect how the status code is returned; its orthogonal.

Does that make sense?